user/topics/data/filesystem.md

```diff
@@ -13,12 +13,29 @@ However, you can’t ssh in!
 Your username is ``jovyan``, and your home directory is ``/home/jovyan``.
 This is the same for all users, but no one else can see or access the files in *your* home directory.
 
-``/home/jovyan`` is a persistant network-attached drive. Any files you put there will be there when you
-log out and log back into the JupyterHub. 
+``/home/jovyan`` is a persistent network-attached drive. Any files you put there will be there when you
+log out and log back into the JupyterHub.
 
-The ``/home/jovyan`` space is intended only for notebooks and code. It's not an appropriate place to store
+The ``/home/jovyan`` space is intended only for notebooks and code. It's **not** an appropriate place to store
 datasets, as it can get really expensive (and slow) when used that way.
 
+### Per-User Storage Quotas
+
+All of our hubs have a 10GB storage quota per-user by default, although this may vary depending on the hub.
```

Comment:
@jnywong This isn't technically true as we have not rolled jupyterhub-home-nfs across all AWS hubs yet.

Reply:
Yeah this has always been tricky with writing docs for the Service Guide because a lot of our hubs are "custom" in some way. But I am taking a forward-facing look given all the work we are doing with standardisation in our new "Product Menu 2.0".

> ... although this may vary depending on the hub.

is hopefully enough of a caveat for the reader, with further advice down below to contact hub admins if they run into any problems.

Happy to revise the wording if this is still insufficient for you!